tools/dotcms-cli/cli/src/main/java/com/dotcms/api/client/util/NameUtil.java

```diff
@@ -0,0 +1,47 @@
+package com.dotcms.api.client.util;
+
+import com.dotcms.contenttype.model.type.ContentType;
+import com.dotcms.model.language.Language;
+import com.dotcms.model.site.SiteView;
+
+/**
+ * The NameUtil class provides utility methods for generating file names based on different
+ * objects.
+ */
+public class NameUtil {
```

Comment:
I'd say NamingUtils is probably a better name for this class

Reply:
Thanks, yeah, it is a better name for sure, done.